### DISCUSSION TITLE
NASA CRM Mesh and Config Files

### DISCUSSION THREAD

**Discussion by UrsaMajorrr** [su2code/SU2#2395] [General]
Does anyone have a mesh and config file for the NASA CRM that converges reliably? Preferably one that is already parameterized for FFD boxes

**Comment by bigfooted** [comment]
Here are the latest meshes, you can use the CFD solver with CGNS meshes directly, and convert them to SU2 meshes:
 https://hiliftpw.larc.nasa.gov/Workshop5/grids_downloads_case1.html
